powerlevel9k.zsh-theme

```diff
@@ -744,9 +744,11 @@ prompt_docker_machine() {
 # GO prompt
 prompt_go_version() {
   local go_version
+  local go_path
   go_version=$(go version 2>/dev/null | sed -E "s/.*(go[0-9.]*).*/\1/")
+  go_path=$(go env GOPATH)
 
-  if [[ -n "$go_version" ]]; then
+  if [[ -n "$go_version" && "${PWD##$go_path}" != "$PWD" ]]; then
```

Comment:
This removes the longest submatch of `GOPATH` from `PWD`, and returns true if the remainder of the truncation is not equal to `PWD`. If `GOPATH` is empty, then nothing is truncated from `PWD`, and the go version is not printed..